### DISCUSSION TITLE
能否提供一个完整的UMI4 的插件例子

### DISCUSSION THREAD

**Discussion by haobarry** [umijs/umi#11006] [Q&A] [answered]
如题，最好是有如何导出，如何注册，如何使用。我看了官方文档介绍都是一段一段的，特别是插件的使用, 对于一个刚接触umi的新手不太清楚，可能umi老手比较熟悉

**Comment by fz6m** [comment, marked as answer]
直接参考 `@umijs/plugins` 内的源码即可。

目前确实是缺少从 0 到 1 插件编写的文档、文章等，有待规划和补充。
